httpbakery/form/form_test.go

```diff
@@ -256,12 +306,53 @@ var testError = &httpbakery.Error{
 	Message: "test error",
 }
 
-type fillerFunc func(environschema.Fields) (map[string]interface{}, error)
+type fillerFunc func(form.Form) (map[string]interface{}, error)
 
-func (h fillerFunc) Fill(f environschema.Fields) (map[string]interface{}, error) {
+func (h fillerFunc) Fill(f form.Form) (map[string]interface{}, error) {
 	return h(f)
 }
 
-var defaultFiller = fillerFunc(func(environschema.Fields) (map[string]interface{}, error) {
+var defaultFiller = fillerFunc(func(form.Form) (map[string]interface{}, error) {
 	return map[string]interface{}{"test": 1}, nil
 })
+
+type testLocator struct {
+	loc     string
+	locator bakery.PublicKeyLocator
+}
+
+func (l testLocator) PublicKeyForLocation(loc string) (*bakery.PublicKey, error) {
+	return l.locator.PublicKeyForLocation(l.loc)
+}
+
+type titleTestFiller struct {
+	title string
+}
+
+func (f *titleTestFiller) Fill(frm form.Form) (map[string]interface{}, error) {
+	f.title = frm.Title
+	return map[string]interface{}{"test": 1}, nil
+}
+
+type transport struct {
```

Comment:
A doc comment here would be useful - I spent a little time trying to work out what this was actually doing.

// transport implements an HTTP transport that wraps rt
// and replaces the specified prefix string in outgoing request
// addresses with the specified address.

?